simapp/simd/cmd/root.go

```diff
@@ -263,52 +256,13 @@ func newApp(
 	traceStore io.Writer,
 	appOpts servertypes.AppOptions,
 ) servertypes.Application {
-	var cache sdk.MultiStorePersistentCache
 
-	if cast.ToBool(appOpts.Get(server.FlagInterBlockCache)) {
-		cache = store.NewCommitKVStoreCacheManager()
-	}
-
-	skipUpgradeHeights := make(map[int64]bool)
```

Comment:
This is sad that we have no tooling to catch these. I thought golanglint was smart enough to see this.

Reply:
hmm this seems like it was a bug, it was never used?

Reply:
Yes this is what I meant. It wasn't used and no tooling catched that before you.

Reply:
A linter wouldn't catch this as `skipUpgradeHeights` technically is used (values being set). A smarter tool would be needed.